guides/common/modules/proc_uploading-ostree-content-with-hammer-cli.adoc

```diff
@@ -0,0 +1,34 @@
+[id="uploading-ostree-content-with-hammer-cli_{context}"]
+= Uploading OSTree Content with Hammer CLI
+
+In deployments in which the {ProjectServer} does not have internet access, you can upload OSTree image archives using Hammer CLI.
+
+You can create images using the Image Builder or OSBuild tool.
+
+You can also create an OSTree image archive by downloading a repository from an upstream URL, such as:
+[options="nowrap" subs="+quotes,attributes"]
+----
+wget --no-parent -r _https://repos.example.com/ostree/repo_name/_
+tar --exclude="index.html" -cvf "_example_archive_repo_.tar" -C _repos.example.com/ostree_ "_repo_name_"
+----
+
+.Prerequisite
+* You have an OSTree image archive ready and present on your file system.
+
+.Procedure
+. Connect to your {ProjectServer} using SSH.
+. Upload your OSTree archive with the `hammer` command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+hammer repository upload-content --id _repository_id_ \
+--content-type ostree_ref \
+--path _/path/to/image/file.tar_ \
+--ostree-repository-name _example_repo_name_
+----
++
+The value of `--ostree-repository-name` must match the name of the OSTree repository in the archive.
+
+.Additional resources
+* https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/composing_installing_and_managing_rhel_for_edge_images/index[Composing, installing, and managing RHEL for Edge images using Image Builder]
```

Comment:
Can you please make this line non-orcharhino only?

Reply:
Sure!
Thanks for all your catches. On it.